src/components/ha-dialog.ts

```diff
@@ -90,7 +90,7 @@ export class HaDialog extends DialogBase {
       }
       .mdc-dialog__actions {
         justify-content: var(--justify-action-buttons, flex-end);
-        padding-bottom: max(env(safe-area-inset-bottom), 24px);
+        padding: 12px 24px max(env(safe-area-inset-bottom), 24px) 24px;
```

Comment:
```suggestion
        padding: 12px 24px max(env(safe-area-inset-bottom), 12px) 24px;
```
Shouldn't this be 12px?